badges-metadata/main/hydra-s1-accountbound.ts

```diff
@@ -885,6 +885,11 @@ export const hydraS1AccountboundBadges: BadgesCollection = {
           contact: "zk-pay",
         },
       ],
+      curatedAttributes: {
```

Comment:
The curatedAttributes are added after the badge curation process https://www.notion.so/sismo/ZK-Badge-Curation-Guidelines-a62b5d7205734b44b41c603bf1516f4c
Can you remove them for now?